tests/unit/libbpf_test.cpp

```diff
@@ -495,41 +494,6 @@ test_xdp_ifindex(uint32_t ifindex, int program_fd[2], bpf_prog_info program_info
     REQUIRE(errno == ENOENT);
 }
 
-static void
-_test_bpf_set_link_xdp_fd(ebpf_execution_type_t execution_type)
```

Comment:
since this is testing the libbpf xdp APIs, I think we should keep these tests in unit tests (and also keep the mock XDP implementation for unit tests)

Reply:
The Idea was to remove all XDP test programs and also the XDP definitions. I will file a work item to bring back the mock XDP provider if everyone agrees,

Reply:
https://github.com/microsoft/ebpf-for-windows/issues/4706